### ISSUE TITLE
loss function in "logistic-regression.ipynb "

### ISSUE THREAD

**Issue by wanpiqiu123** [SherlockLiao/code-of-learn-deep-learning-with-pytorch#37]
计算loss的函数：
```
def binary_loss(y_pred, y):
    logits = (y * y_pred.clamp(1e-12).log() + (1 - y) * (1 - y_pred).clamp(1e-12).log()).mean()
    return -logits
```
前半部分是不是应该使用平均值？即
```
logits = (y * y_pred.clamp(1e-12).log().mean() + (1 - y) * (1 - y_pred).clamp(1e-12).log()).mean()
```

**Comment by L1aoXingyu** [comment]
均值的括号是在最外面，不是只对最后一个部分求mean

**Comment by wanpiqiu123** [comment]
感谢！
